### ISSUE TITLE
Unhandled Error not filtered

### ISSUE THREAD

**Issue by chrmarti** [Microsoft/vscode#16683]
- VSCode Version: Code - Insiders 1.8.0-insider (7499a9f3c565f4fbe5205e30c8664e5a5a11ae94, 2016-12-06T16:10:17.195Z)
- OS Version: Darwin x64 15.6.0
- Extensions:

|Extension|Author|Version|
|---|---|---|
|EditorConfig|EditorConfig|0.3.4|
|vscode-docker|PeterJausovec|0.0.7|
|regex|chrmarti|0.0.7|
|vscode-eslint|dbaeumer|1.1.0|
|ghlink|dt|0.0.4|
|tslint|eg2|0.6.0|
|PowerShell|ms-vscode|0.7.2|
|debugger-for-chrome|msjsdiag|2.2.2|
|diff|rafaelmaiolla|0.0.1|
|vscode-open-in-github|sysoev|1.2.1|

---

Testing #16208

Steps to Reproduce:

1. launch.json:
```
{
    "version": "0.2.0",
    "configurations": [
        {
            "type": "node",
            "request": "launch",
            "name": "Launch Program",
            "program": "${workspaceRoot}/test.js",
            "skipFiles": [
                "*.js"
            ],
            "cwd": "${workspaceRoot}"
        }
    ]
}
```
2. test.js:
```
throw new Error('err');
```

It works as expected with `node2`.

**Comment by chrmarti** [comment]
It also works as expected with `**/*.js` and `node`.

**Comment by chrmarti** [comment]
@isidorn Are you using glob.ts?

**Comment by roblourens** [comment]
I guess your typical glob would require `**/*.js` but node2 allows globs for filenames or paths, because of the way it's implemented in chrome.

**Comment by weinand** [comment]
@chrmarti no, I'm using "glob" and "minimatch".

**Comment by weinand** [comment]
Yes, the correct glob pattern is `**/*.js`. So this works as designed. But I will reconsider this in the next milestone.

**Comment by FishOrBear** [comment]
@weinand 
It may be better to set the global configuration and local configuration , otherwise set the same `skip file" for different configurations.
